app/controllers/index.rb

```diff
@@ -0,0 +1,66 @@
+get '/' do
+ redirect '/suggestions'
+end
+
+get '/suggestions' do
+  @suggestions = Suggestion.all
+  erb :'suggestions/index'
+end
+
+get '/suggestions/new' do
+  # @user = User.find_by(params[:id])
+  # if @user.valid?
+  #   redirect '/'
+  # else
+  #   status 422
+  #   @errors = user.errors.full_messages
+  #   erb :'suggestions/new'
+  # end
+
+  if !logged_in?
+    redirect '/suggestions'
+  else
+    erb :'suggestions/new'
+  end
+end
+
+post '/suggestions' do
+  # binding.pry
+  if logged_in?
+    @suggestion = Suggestion.create(params[:suggestion])
+    redirect '/'
+  else
+    erb :'sugestions/new'
+  end
+end
+
+get '/suggestions/:id' do
+  # binding.pry
+  @user = User.find(params[:id])
+  @suggestion = Suggestion.find(params[:id])
+   if !logged_in?
+    redirect '/suggestions'
+  else
+    @error = 'No Dice!'
+    erb :'suggestions/show'
```

Comment:
Looks like you are always passing an error message to the suggestion show page.  Only pass an error if the user should not see this page.